src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -94,36 +94,14 @@ export default class GeneSetSelector extends QueryStoreComponent<{}, {}>
 					onChange={(option:any) => this.store.geneQuery = option ? option.value : ''}
 				/>
 
-				{/* we are hiding these buttons on 02/27/2019 */}
-				{/* {!!(this.store.mutSigForSingleStudy.result.length || this.store.gisticForSingleStudy.result.length) && (
-					<FlexRow padded className={styles.buttonRow}>
-						{!!(this.store.mutSigForSingleStudy.result.length) && (
-							<button className="btn btn-default btn-sm" onClick={() => this.store.showMutSigPopup = true}>
-								Select from Recurrently Mutated Genes (MutSig)
-							</button>
-						)}
-						{!!(this.store.gisticForSingleStudy.result.length) && (
-							<button className="btn btn-default btn-sm" onClick={() => this.store.showGisticPopup = true}>
-								Select Genes from Recurrent CNAs (Gistic)
-							</button>
-						)}
-					</FlexRow>
-				)} */}
-
-				<textarea
-					ref={this.textAreaRef}
-					className={classNames(styles.geneSet, this.store.geneQuery ? styles.notEmpty : styles.empty)}
-					rows={5}
-					cols={80}
-					placeholder="Enter HUGO Gene Symbols, Gene Aliases, or OQL"
-					title="Enter HUGO Gene Symbols, Gene Aliases, or OQL"
-					value={this.store.geneQuery}
-					onChange={event => this.store.geneQuery = event.currentTarget.value}
-					data-test='geneSet'
+				<GeneSelectionBox // misleading: this is where you type genes / oql. Select is the ReactSelect
```

Comment:
Good point.  Lets change it!  OQLInput? OQLTextArea?

Reply:
OQLTextArea works for me